src/Sylius/Behat/Context/Api/Admin/ManagingTaxonsContext.php

```diff
@@ -95,20 +95,13 @@ public function iShouldSeeTaxonsInTheList($number)
      * @Then I should see the taxon named :firstName, :secondName and :thirdName in the list
      * @Then I should see the taxon named :firstName, :secondName, :thirdName and :fourthName in the list
      */
-    public function iShouldSeeTheTaxonNamedAnd(... $taxonNames)
+    public function iShouldSeeTheTaxonNamedAnd(...$expectedTaxonNames)
     {
-        foreach ($taxonNames as $name) {
-            $response = json_decode($this->client->getResponse()->getContent(), true);
-
-            $taxonNames = array_map(function ($item) {
-                return $item['name'];
-            }, $response);
+        $response = json_decode($this->client->getResponse()->getContent(), true);
+        $taxonNames = array_map(function ($item) {
+            return $item['name'];
+        }, $response);
 
-            if (!in_array($name, $taxonNames)) {
-                throw new \InvalidArgumentException(
-                    sprintf('There should be taxon named "%s", but got only "%s"', $name, $this->client->getResponse()->getContent())
-                );
-            }
-        }
+        Assert::allOneOf($taxonNames, $expectedTaxonNames);
```

Comment:
Shouldn't the order of arguments be reverted? We are looking for each if `$expectedTaxonNames` in `$taxonNames` collection.